maven-build-scan/load/action.yml

```diff
@@ -0,0 +1,68 @@
+name: Load Maven Build Scans
+description: Load Maven Build Scans
+
+inputs:
+  build-workflow-filename:
+    description: 'Filename of the workflow where the maven-build-scan/save action was triggered'
+    required: true
+  pr-comment-tos-acceptance-request:
+    description: 'pull-request comment to accept the Terms of Service'
+    required: true
+
+outputs:
+  pr-number:
+    description: "pull-request number"
+    value: ${{ steps.pr.outputs.PR_NUMBER }}
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Check event trigger
+      if: |
+        (github.event_name != 'issue_comment' 
+          || (
+            github.event.comment.body != 'recheck' 
+            && github.event.comment.body != inputs.pr-comment-tos-acceptance-request
+          )
+        )
+        && github.event_name != 'workflow_run'
+      run: |
+        echo "Skipping Github event"
```

Comment:
This message won't make much sense in the logs on it's own, I don't think. What event is being skipped and why? (Also, maybe this should be logged as DEBUG)

Reply:
I adjusted the message, I kept it in INFO as I think adding some feedback makes the output clearer